### ISSUE TITLE
Fix the oldest plmenu plugin bug

### ISSUE THREAD

**Issue by all85100** [alliedmodders/amxmodx#563]
Fix the oldest plmenu plug-in bug, when using Slap slay menu function, give the player multiple 5 damage, when the player's blood is not enough 5 o'clock, should be executed directly the player! It's more reasonable.

**Comment by Arkshine** [comment]
I think it's by design it doesn't kill the player. You can easily spam the slap without wanted to kill the player and in a blink of an eye, the player could be dead.  I might be wrong but I think it's better to not allow slap to slay if no more health.
That's said, I don't play anymore, maybe people would prefer this way. I would like to hear others thoughts on that.

**Comment by OciXCrom** [comment]
I agree it's better as it is right now. If you want to slay the player, you can easily switch to the `slay` option in the same menu. Changing the behavior like you suggested it will do more harm than good.
